pkg/pillar/hypervisor/kvm.go

```diff
@@ -462,20 +462,135 @@ func (ctx kvmContext) Task(status *types.DomainStatus) types.Task {
 	}
 }
 
```

Comment:
Typo in commit message: 
`E.g. NVIDIA A40 has 64 Gb apertura` , "apertura" -> "aperture"

Reply:
fixed